nevergrad/optimization/optimizerlib.py

```diff
@@ -2275,6 +2276,40 @@ def __init__(
 NaiveIsoEMNA = EMNA().set_name("NaiveIsoEMNA", register=True)
 
 
+try:
+    from modcma import AskTellCMAES
+
+    modcma_available = True
+except ImportError:
+    modcma_available = False
+
+if modcma_available:
```

Comment:
please just do the import inside the `__init__`

Reply:
Done.